frameworks/kafka/tests/utils/kafka_networks.py

```diff
@@ -0,0 +1,28 @@
+"""
+Kafka-Specific Utilities on networking
+
+(This file contains the kafka-specific additions on the `sdk_networks` file)
+"""
+import shakedown
```

Comment:
We have [deprecated shakedown](https://github.com/mesosphere/dcos-commons/commit/7f8028d37231a32df5e3f091eaa9b4934456844c) in dcos-commons. Should we do the same here?

Reply:
Done :+1: